appendices/symbol-lookup-cheatsheet.md

```diff
@@ -41,7 +42,7 @@ Here is a more elaborate explanation of Pony's use of special characters:
 | { }  | a function type
 | :  | (1) after a variable: is followed by the type name 
      | (2) to indicate a function return type
-|    | (3) a type constraint
+     | (3) a type constraint
```

Comment:
No, I will put leading | for all entries.

 

From: Sean T Allen [mailto:notifications@github.com] 
Sent: maandag 11 april 2016 17:09
To: ponylang/pony-tutorial <pony-tutorial@noreply.github.com>
Cc: ibalbaert <ivo.balbaert@telenet.be>
Subject: Re: [ponylang/pony-tutorial] Added 2nd usage section for symbol cheat-sheet (#91)

 

In appendices/symbol-lookup-cheatsheet.md <https://github.com/ponylang/pony-tutorial/pull/91#discussion_r59221394> :

> @@ -41,7 +42,7 @@ Here is a more elaborate explanation of Pony's use of special characters:
>  | { }  | a function type
>  | :  | (1) after a variable: is followed by the type name 
>       | (2) to indicate a function return type
> -|    | (3) a type constraint
> +     | (3) a type constraint

The mixing of use of leading and non-leading | as option strikes me as error prone.

Any reason not to have the leading | for all entries?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub <https://github.com/ponylang/pony-tutorial/pull/91/files/80196091b4e62a5b9b923bb290ab4aac68909f35..97dd908382905cf61f37b871b3990982fcdbab02#r59221394>   <https://github.com/notifications/beacon/AA6zfI8JdHf7tMZnAbAlxMmDYr7wqRcpks5p2mQMgaJpZM4IEQOw.gif> 



---
Dit e-mailbericht is gecontroleerd op virussen met Avast antivirussoftware.
https://www.avast.com/antivirus